src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/CSharpEESymbolProvider.cs

```diff
@@ -102,8 +102,7 @@ public override IAssemblySymbolInternal GetReferencedAssembly(AssemblyReferenceH
         /// <exception cref="UnsupportedSignatureContent"></exception>
         public override TypeSymbol GetType(EntityHandle handle)
         {
-            bool isNoPiaLocalType;
-            return _metadataDecoder.GetSymbolForTypeHandleOrThrow(handle, out isNoPiaLocalType, allowTypeSpec: true, requireShortForm: false);
+            return _metadataDecoder.GetSymbolForTypeHandleOrThrow(handle, out _, allowTypeSpec: true, requireShortForm: false);
```

Comment:
the EE is not under the compiler and doesn't ahve its own editorconfig.  So i did fix these all up.  THere were few enough cases that i felt ok doing this.